Cargo.toml

```diff
@@ -8,7 +8,7 @@ repository = "https://github.com/georust/geojson"
 readme = "README.md"
 documentation = "https://docs.rs/geojson/"
 keywords = ["geojson", "gis", "json", "geo"]
-edition = "2018"
```

Comment:
Was there a specific cause to bump this edition? It's probably fine for us to move forward, but it's good to know if there's a reason for it since it can affect downstream users.

And it should probably go in the changelog.

Reply:
None that were warranted for this PR. The new edition includes the `TryInto` traits in the prelude 😅🙈 I'll go ahead and revert this immediately, apologies.